.github/workflows/py3.yml

```diff
@@ -22,12 +22,11 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
-      - name: Install pynetbox and testing packages.
-        run: pip install . black==19.10b0 pytest pytest-docker faker
+      - name: Install dev requirements
+        run: pip install -r requirements-dev.txt .
```

Comment:
Nice.. I thought requirements files and individual module names had to be ran via separate commands which is why I had separated these initially but this works just fine. Thanks for the pointer

Reply:
Yep, thought the same thing. Gave it a shot though and it seemed to work. :man_shrugging: